packages/lib/components/shared/dropbox-login-shared.js

```diff
@@ -1,6 +1,6 @@
 const shim = require('../../shim').default;
 const SyncTargetRegistry = require('../../SyncTargetRegistry').default;
-const { reg } = require('../../registry.js');
+const { reg } = require('../../registry');
```

Comment:
Using `registry.js` here was causing **two copies** of `registry` to be included — the copy from files that `require('registry')` and the copy from files that `require('registry.js')`. As a result, certain global variables/properties in `registry` were not initialized in one copy, but were in the other.

To work around this issue, this pull request changes all `registry.js` imports to `registry` imports. Perhaps using [`metro` instead of `webpack`](https://github.com/necolas/react-native-web/issues/1257) would be a different way to address this issue?